L/Libgcrypt/build_tarballs.jl

```diff
@@ -33,8 +40,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Libgpg_error_jll", v"1.42.0"; compat="1.42.0"),
+    Dependency("Libgpg_error_jll", v"1.50"; compat="1.49"),
```

Comment:
Note that the lowest version of the compat is automatically used as build version, and if specifying a build version is important, having a consistent compat is usually important too
```suggestion
    Dependency("Libgpg_error_jll"; compat="1.50"),
```

Reply:
Thing is, the compat really is 1.49, we just only have a build for FreeBSD-aarch64 for 1.50.